turbo/src/main/kotlin/dev/hotwire/turbo/visit/TurboVisitError.kt

```diff
@@ -0,0 +1,18 @@
+package dev.hotwire.turbo.visit
+
+data class TurboVisitError(
```

Comment:
As an improvement, we could also use a sealed interface with subtypes only containing the properties that they use instead of a data class with optional or unused properties.

Something like:

```kotlin
sealed interface TurboVisitError

data class LoadTurboVisitError(val description: String?) : TurboVisitError
data class HttpTurboVisitError(val code: Int, val description: String?) : TurboVisitError
data class WebResourceTurboVisitError(val code: Int, val description: String?) : TurboVisitError
data class WebSslTurboVisitError(val code: Int, val description: String?) : TurboVisitError
```

Reply:
Sorry, I forgot to update that I talked to @jhutarek yesterday and we arrived at the same conclusion. Working on updating that now :)